rfc/cdevents-translator.md

```diff
@@ -0,0 +1,41 @@
+## CDEvents Translator
+
+### Overview:
+This proposal describes translating events from various source control systems into CDEvents and sending them to configured message-broker, by using a common cdevents-translator 
+
+A library cdevents-translator will be developed to receive various event types and convert them into CDEvents.
```

Comment:
How this will work in terms of supporting multiple types?
Will the library provide an interface that various implementors can implement independently?

Reply:
yes an interface `EventTranslator` will be created with the implementation needed for `TranslateEvent()` for different source of events(GitHub/Gerrit`/...etc)

````go
type EventTranslator interface {
	TranslateEvent()
}
````
And will have a common methods to create and send CDEvents, will update the design as per this.